src/Tests/Unit/SAML2/ReceivedAuthnRequestQueryStringTest.php

```diff
@@ -46,34 +48,21 @@ class ReceivedAuthnRequestQueryStringTest extends TestCase
      * @dataProvider emptyStringProvider
      * @param $nonOrEmptyString
      */
-    public function a_received_authn_request_query_string_cannot_be_parsed_from_empty_strings($nonOrEmptyString): void
+    public function a_received_authn_request_query_string_cannot_be_parsed_from_empty_strings(string $nonOrEmptyString): void
     {
-        $this->expectException('\Surfnet\SamlBundle\Http\Exception\InvalidReceivedAuthnRequestQueryStringException');
+        $this->expectException('\\' . InvalidReceivedAuthnRequestQueryStringException::class);
```

Comment:
Why the two slashes in front?

Reply:
On escapes the backslash that is used to get an absolute fqn reference to the class that is expected. This is not required afaic. Removed these constructions